### DISCUSSION TITLE
I don't have an objects menu when I drag media into a room

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#2789] [General Questions & Discussion] [answered]
I see on the documentation that when an object is brought into a space, there should be a menu where I can pin/delete the object.  I test uploaded a pdf, and I don't see any such menu.  I see the snapshot option when highlighting the pdf, along with page buttons--but that's all.  How do I bring up the "pin" option?  I have pin objects enabled for this room.

**Comment by emclaren** [comment, marked as answer]
Hi there! good question. If you are on desktop you should be able to point at the object and press the space bar, or tab key to open up the hidden menus on an object. If you are on a mobile device you can use a two finger tap. If you are on a VR headset it is one of the buttons on the controller (see https://hubs.mozilla.com/docs/hubs-controls.html)

**Reply by hilarytsai** [reply]
Thank you @emclaren!  This worked, I would never have known otherwise.  Appreciate you!
